fastlane/Helpers/Distribution.swift

```diff
@@ -25,4 +25,27 @@ enum Distribution {
             debug: .userDefined(true)
         )
     }
+
+    static func uploadToAppStore() {
+        let environment: Constant.Environment = .production
+        let ipaPath = makeIpaPath(environment: environment)
+        appStoreConnectApiKey(
+            keyId: Constant.appStoreKeyId,
+            issuerId: Constant.appStoreIssuerId,
+            keyContent: .userDefined(Secret.appstoreConnectAPIKey),
+            isKeyContentBase64: .userDefined(true) // Check if the AppStore Connect API Key is base64
```

Comment:
```suggestion
            isKeyContentBase64: .userDefined(true) // Use AppStore Connect API Key in base64 encoded
```

Reply:
@blyscuit We have two ways to procedure API key content, so I think it's better if the user checks to ensure they're using base64 or not. We don't need to force using base64 format, IMO.